### PR TITLE
blog(embertimes170): octane mental model

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 tags: Recent Posts, Newsletter, Performance, Ember Octane, Ember.js Times, Ember Times, 2020
 alias : "blog/2020/12/18-the-ember-times-issue-170.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-Explore Ember/Glimmer component performance 🚅, .
+Explore Ember/Glimmer component performance 🚅,
+Read the blog post on the Ember Octane mental model ✍️,
```
Comment (amyrlam) [inline source/2020-12-18-the-ember-times-issue-170.md:13]:
```suggestion
<!--alex ignore nuts-->
Read the blog post on the Ember Octane mental model ✍️,
```

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 2]
```diff
@@ -28,14 +29,13 @@ Within [the source README.md](https://github.com/NullVoxPopuli/ember-three-boxes
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Blog: Ember Octane is a New Mental Model ✍️](https://v5.chriskrycho.com/journal/ember-octane-is-a-new-mental-model/)
```
Comment (amyrlam) [inline source/2020-12-18-the-ember-times-issue-170.md:34]:
```suggestion
<!--alex ignore nuts-->
## [Blog: Ember Octane is a New Mental Model ✍️](https://v5.chriskrycho.com/journal/ember-octane-is-a-new-mental-model/)
```

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 3]
```diff
@@ -28,14 +29,13 @@ Within [the source README.md](https://github.com/NullVoxPopuli/ember-three-boxes
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) wrote a [blog post](https://v5.chriskrycho.com/journal/ember-octane-is-a-new-mental-model/) on how Ember Octane is not a 1:1 translation from Ember Classic but rather a whole new way of thinking and designing your code. This new programming model means that some patterns that you’re used to using in the Classic paradigm simply do not work in the new paradigm.
```
Comment (amyrlam) [inline source/2020-12-18-the-ember-times-issue-170.md:37]:
```suggestion
<!--alex ignore easy-->
[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) wrote a [blog post](https://v5.chriskrycho.com/journal/ember-octane-is-a-new-mental-model/) on how Ember Octane is not a 1:1 translation from Ember Classic but rather a whole new way of thinking and designing your code. This new programming model means that some patterns that you’re used to using in the Classic paradigm simply do not work in the new paradigm.
```

This didn't fail in CI but I think it does, cause I see the error in the alex extension on VSCode

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 4]
```diff
@@ -28,14 +29,13 @@ Within [the source README.md](https://github.com/NullVoxPopuli/ember-three-boxes
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Chris goes through a couple examples of code that was fundamentally coupled to Ember Classic idioms such as the [Evented API](https://api.emberjs.com/ember/release/classes/Evented) and usage of `didReceiveAttrs`. The blog argues these cases where rare, it’s possible the abstraction would benefit from being reworked substantially towards the Octane idioms.
```
Comment (amyrlam) [inline source/2020-12-18-the-ember-times-issue-170.md]:
```suggestion
Chris goes through a couple examples of code that was fundamentally coupled to Ember Classic idioms such as the [Evented API](https://api.emberjs.com/ember/release/classes/Evented) and usage of `didReceiveAttrs`. The blog argues that although these cases were rare, it’s possible the abstraction would benefit from being reworked substantially towards the Octane idioms.
```

Did you mean `were`? Wasn't sure.

**Review by amyrlam — APPROVED** [review]
Thank you for covering this! 🙇‍♀️ 

We are on a roll, getting things up early, much appreciated! 📈

File: source/2020-12-18-the-ember-times-issue-170.md (hunk 4 above)
Comment (chrisrng) [reply source/2020-12-18-the-ember-times-issue-170.md]: Yikes yes :D

**Pull request merged**